docs/start/hello-world.ipynb

```diff
@@ -7,23 +7,42 @@
    "source": [
     "# Hello world\n",
     "\n",
-    "This Hello world example creates a simple quantum program and runs it on a quantum system. Begin with following the [Install and set up](install) instructions if you haven't already, including the steps to [Set up to use IBM Quantum&trade; Platform](setup-channel#set-up-to-use-ibm-quantum-platform).\n",
+    "This example contains two parts: you will first create a simple quantum program and run it on a quantum system.  Because actual quantum research requires much more robust programs, in the [Scale to large numbers of qubits](#scale-to-large-numbers-of-qubits) section, you will scale the simple program up to utility level.  You can also follow along with the Hello World episode of the Coding with Qiskit 1.0 video series. \n",
     "\n",
-    "We recommend that you use the [Jupyter](https://jupyter.org/install) development environment to interact with quantum computers. Be sure to install the recommended extra visualization support (`pip install qiskit[visualization]`), and note that zsh users need to put `'qiskit[visualization]'` in single quotes.\n",
+    "ame width=\"560\" height=\"315\" src=\"https://www.youtube.com/embed/93-zLTppFZw?si=whJzB_FrpuorerTv\" title=\"Hello World\" frameborder=\"0\" allow=\"accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture\" allowfullscreen></iframe>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7b65f7e0",
+   "metadata": {},
+   "source": [
+    "## Before you begin\n",
     "\n",
-    "To learn about quantum computing in general, check out the [Basics of quantum information course](https://learning.quantum.ibm.com/course/basics-of-quantum-information) in IBM Quantum Learning.\n",
+    "Follow the [Install and set up](install) instructions if you haven't already, including the steps to [Set up to use IBM Quantum&trade; Platform](setup-channel#set-up-to-use-ibm-quantum-platform).\n",
     "\n",
-    "The four steps to writing a quantum program are\n",
+    "It is recommended that you use the [Jupyter](https://jupyter.org/install) development environment to interact with quantum computers. Be sure to install the recommended extra visualization support (`pip install qiskit[visualization]`), and note that zsh users need to put `'qiskit[visualization]'` in single quotes. You'll also need the `matplotlib` package for the second part of this example.\n",
     "\n",
-    "1.  Map the problem to a quantum-native format\n",
     "\n",
-    "2.  Optimize the circuits and operators\n",
+    "To learn about quantum computing in general, check out the [Basics of quantum information course](https://learning.quantum.ibm.com/course/basics-of-quantum-information) in IBM Quantum Learning.\n",
```

Comment:
"check out" seems not translation-friendly, though I'm having trouble finding a good equivalent. "visit"? "explore"?